src/main/java/org/folio/circulation/resources/CirculationRulesResource.java

```diff
@@ -115,14 +147,78 @@ private void put(RoutingContext routingContext) {
     LoanCirculationRulesEngineResource.clearCache(new WebContext(routingContext).getTenantId());
     RequestCirculationRulesEngineResource.clearCache(new WebContext(routingContext).getTenantId());
 
-    loansRulesClient.put(rulesInput.copy()).thenAccept(result ->
-      result.applySideEffect(response -> {
+    clients.circulationRulesStorage().put(rulesInput.copy())
+      .thenAccept(res -> res.applySideEffect(response -> {
         if (response.getStatusCode() == 204) {
           SuccessResponse.noContent(routingContext.response());
         } else {
           ForwardResponse.forward(routingContext.response(), response);
         }
-    }, cause -> cause.writeTo(routingContext.response())));
+      }, cause -> cause.writeTo(routingContext.response())));
+  }
+
+   private boolean policyExists(Map<String, Set<String>> existingPolicyIds,
+                                String policyType, List<CirculationRulesParser.PolicyContext> policies) {
```

Comment:
Please can this be formatted with the standard indentation rather than horizontal alignment